mod-inventory/src/main/resources/domain/mod-inventory/features/inventoryFeatureTest.feature

```diff
@@ -0,0 +1,12 @@
+Feature: inventory sample
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+    
+
+  Scenario: print
```

Comment:
This should have an `@Undefined` annotation as should any further "test stubs" (specified but unimplemented tests). I believe adding this annotation to scenarios will allow the scenario to appear in the Cucumber reports as undefined (i.e. work which needs to be done) which is useful to the team overseeing the karate tests.

Reply:
Here's a link in the docs about the use of this `@Undefined` annotation: https://wiki.folio.org/display/FOLIJET/Karate+test+plan+for+cucumber+report